utils/checks/features

```diff
@@ -77,6 +77,10 @@ def list_expected_propagated_features(feature, _package, deps)
     default_enabled = dep[:dependency]['uses_default_features']
     next false if default_enabled && package_feature_enabled_by(dep[:package], 'default', feature)
 
+    # If the package is used as dev dependency - skip it, as we don't need to specify
+    # the feature.
+    next false if dep[:dependency]['kind'] == 'dev'
```

Comment:
To fix - https://github.com/humanode-network/humanode/actions/runs/4786888597/jobs/8511395058?pr=664 as https://github.com/rust-lang/cargo/issues/7754.

Reply:
Right, but it's only when the feature is `std` actually, as we might want to propagate other features properly (i.e. runtime benchmarks and etc).